src/test/kotlin/org/jetbrains/research/ml/coding/assistant/util/PsiTestUtil.kt

```diff
@@ -47,7 +45,7 @@ object PsiTestUtil {
             return false
         }
         // Compare labels
-        if (psi.label != tree.label) {
+        if (psi.getLabel() != tree.label) {
```

Comment:
Why did you change the property to the function? I think the old version is better...

Reply:
Because it was a field inside the class, and now we don't have such class
I'm not sure we want to have a global property with a label...